lib/encoder/src/Pololu37D.cpp

```diff
@@ -25,3 +25,25 @@ bool Pololu37D::reset() {
   m_QEI.reset();
   return true;
 }
```

Comment:
reset() should update `m_zeroOffsetDeg` and set the encoder position to 0. The idea is that we can zero the encoder at any position

Reply:
done

Reply:
done

Reply:
Actually what I mean is that after we call reset(), we'd expect to see that the encoder output is 0 degrees (assuming it's stationary). The offset should get updated to the true encoder reading each time reset() is called, so afterwards the actual position reading that gets returned is true_encoder_reading - offset.

Reply:
Using the m_QEI.reset() method I'm able to reset the position of the encoder to 0. I can do this without updating the m_zeroOffsetDeg.

Reply:
Oh ok nice, ig in that case we don't need the m_zeroOffsetDeg

Reply:
You should also reset `m_previous_angle_deg`, `m_current_angle_deg`, and `m_speed_deg_per_second` to 0 here

Reply:
ive reset them all